patterns/design-patterns-behavioral/src/main/java/com/baeldung/nulls/UsingOptional.java

```diff
@@ -1,23 +1,50 @@
 package com.baeldung.nulls;
 
+import java.util.List;
 import java.util.Optional;
 
+import static java.util.Collections.emptyList;
+
 public class UsingOptional {
 
-    public Optional<Object> process(boolean processed) {
+    public static final String DEFAULT_VALUE = "Default Value";
 
+    public Optional<Object> process(boolean processed) {
         String response = doSomething(processed);
 
         return Optional.ofNullable(response);
     }
 
-    private String doSomething(boolean processed) {
+    public String findFirst() {
+        return getList().stream()
+            .findFirst()
+            .orElse(DEFAULT_VALUE);
+    }
+
+
+    public Optional<String> findOptionalFirst() {
+        return getList().stream()
+            .findFirst();
+    }
 
+    private List<String> getList() {
+        return emptyList();
+    }
+
+    public Optional<String> optionalListFirst() {
+        return getOptionalList().flatMap(
+                list -> list.stream().findFirst());
```

Comment:
break the line at the `.` of `.flatMap` you can optionally also break the line at the `.` of `findFirst` as well.